shared/fs/banner/system-file-manager-integration-banner/index.tsx

```diff
@@ -180,7 +180,7 @@ const Enabled = (props: Props) => {
 type JustEnabledProps = {onDismiss: null | (() => void)}
 const JustEnabled = ({onDismiss}: JustEnabledProps) => {
   const preferredMountDirs = Container.useSelector(state => state.fs.sfmi.preferredMountDirs)
-  const displayingMountDir = preferredMountDirs.get(0) || ''
+  const displayingMountDir = preferredMountDirs[0] || ''
```

Comment:
Do you still need the `|| ''`? If you do, can this crash if `preferredMountDirs` is empty?

Reply:
I think we do, if not for actual empty array, then at least TSC. If the array is empty, `preferredMountDir[0]` would be `undefined` so it won't crash.